lib/src/credentials.dart

```diff
@@ -28,11 +28,17 @@ enum AuthProviderType {
   /// For authenticating without credentials.
   anonymous,
 
-  _facebook,
-  _google,
-  _apple,
+  /// Authenticate with Facebook account.
+  facebook,
+
+  /// Authenticate with Google account
+  google,
+
+  /// Authenticate with Apple Id
+  apple,
+
```

Comment:
```suggestion
  /// Authenticate with Apple Id
  apple,

  /// Authenticate with Facebook account.
  facebook,

  /// Authenticate with Google account
  google,
  
```

Reply:
The order is important because the enums are matched to core by index.

Reply:
We never actually send the enum to core. I just kept the order as it was. Anyway, it is fine with me.